apps/web/src/components/TextInput/TextInput.js

```diff
@@ -15,43 +14,44 @@ import styles from './TextInput.module.scss'
 // https://facebook.github.io/react/docs/refs-and-the-dom.html#exposing-dom-refs-to-parent-components
 //
 
-export default function TextInput (props) {
-  const {
-    theme = {},
-    onChange,
-    value,
-    inputRef,
-    className,
-    inputClassName,
-    noClearButton,
-    loading,
-    label,
-    internalLabel
-  } = props
+const TextInput = forwardRef(({
+  theme = {},
+  onChange,
+  value,
+  inputRef,
+  className,
+  inputClassName,
+  noClearButton,
+  loading,
+  label,
+  internalLabel,
+  placeholder,
+  onFocus,
+  onBlur,
+  ...props
+}, ref) => {
   const onKeyDown = props.onEnter ? onEnter(props.onEnter) : () => {}
   const otherProps = omit(['onEnter', 'className', 'inputRef', 'theme', 'noClearButton', 'loading', 'label', 'internalLabel', 'inputClassName'], props)
   const clear = () => onChange && onChange({ target: { name: props.name, value: '' } })
 
   const [active, setActive] = useState(false)
 
-  const onBlur = () => { props.onBlur && props.onBlur(); setActive(false) }
```

Comment:
you sure we want to remove this setActive stuff for every TextInput? We use TextInput in a bunch of places

Reply:
i fixed this